trackShape: Oval2
n_agents: 2 
